### ISSUE TITLE
Update on release of Code and Pretrained Weight 

### ISSUE THREAD

**Issue by HashmatShadab** [PKU-YuanGroup/LLaVA-o1#1]

**Comment by xiuyanDL** [comment]
any progress?

**Comment by XuGW-Kevin** [comment]
First of all, thank you very much for your interest in our work!
We plan to provide the pretrained weights within a week and release everything else (dataset, beam search code) in the following weeks. 
This might take longer than expected. However, as an undergraduate student, I also have many classes and homework to manage, so please be patient—I will upload everything as soon as possible!

**Comment by zer0yu** [comment]
Thank you for your detailed update in the issue thread. I really appreciate the effort you’re putting into making the pretrained weights and accompanying resources available, especially while balancing your studies.
I wanted to kindly ask if you could provide more clarity on when the pretrained model might be accessible. I find it really interesting and would love to explore its capabilities. I was wondering if you have any plans or timeline for making this model publicly available for testing or usage.

**Comment by XuGW-Kevin** [comment]
Actually I've mentioned before:

pretrained weights: within a week
others:  in the following weeks.(2-3 weeks)

**Comment by XuGW-Kevin** [comment]
We will also train a model on Qwen2-VL in the coming 2-3 weeks.

**Comment by XuGW-Kevin** [comment]
Pretrained weight: https://huggingface.co/Xkev/Llama-3.2V-11B-cot

**Comment by XuGW-Kevin** [comment]
> @XuGW-Kevin could you clarify if Llama-3.2V-11B-cot is LLaVa-o1 as presented in the paper & benchmarks?  Is there any reason for the different names?

@chris-aeviator It is LLaVA-o1 w/o inference time search. We use the name because Meta requires that all models finetuned from Llama should begin with Llama.

**Comment by fullstackwebdev** [comment]
Hi I am curious if it is possible to get a sample of the dataset thanks

**Comment by ma000-simreka** [comment]
Hi Team LLaVA o1!

Nice work.

I was wondering whether you guys are planning to take up Pixtral model as well? That will be really interesting to see how it comes out.

Best!

**Comment by beniz** [comment]
> We will also train a model on Qwen2-VL in the coming 2-3 weeks.

Hi @XuGW-Kevin thanks for sharing this early. Qwen2-VL handles multiple images, which is very useful in practice, whereas Llama3.2-V doesn't (https://huggingface.co/meta-llama/Llama-3.2-11B-Vision-Instruct/discussions/43).

Happy to help finetuning models once the code is shared.

**Comment by XuGW-Kevin** [comment]
> Hi I am curious if it is possible to get a sample of the dataset thanks

Hi, we're going to release the full dataset soon~

**Comment by XuGW-Kevin** [comment]
> Hi Team LLaVA o1!
> 
> Nice work.
> 
> I was wondering whether you guys are planning to take up Pixtral model as well? That will be really interesting to see how it comes out.
> 
> Best!

Hi, we'll test on Qwen models first. We may move on to Pixtral model if the results on Qwen models are good.

**Comment by XuGW-Kevin** [comment]
1. No, “What's the problem? What should I do?” are hints for readers to understand what the specific stage aims to do. We only use tags like <X></X>. No additional prompts.
2. Yes, in the paper we use the model Llama-3.2V-11B-cot, and the improvements are not significant enough (as you can see from the results). We will explore using a reward model.
